data/102/085/979/102085979.geojson

```diff
@@ -2,251 +2,255 @@
-        "Bedford County"
+    "name:swe_x_preferred": [
+      "Bedford County"
     ],
-    "name:tat_x_preferred":[
-        "\u0411\u0435\u0434\u0444\u043e\u0440\u0434"
+    "name:tat_x_preferred": [
+      "Бедфорд"
     ],
-    "name:tum_x_preferred":[
-        "Bedford County"
+    "name:tum_x_preferred": [
+      "Bedford County"
     ],
-    "name:tur_x_preferred":[
-        "Bedford County"
+    "name:tur_x_preferred": [
+      "Bedford County"
     ],
-    "name:ukr_x_preferred":[
-        "\u0411\u0435\u0434\u0444\u043e\u0440\u0434"
+    "name:ukr_x_preferred": [
+      "Бедфорд"
     ],
-    "name:urd_x_preferred":[
-        "\u0628\u06cc\u0688\u0641\u0648\u0631\u0688 \u06a9\u0627\u0624\u0646\u0679\u06cc\u060c \u0648\u0631\u062c\u06cc\u0646\u06cc\u0627"
+    "name:urd_x_preferred": [
+      "بیڈفورڈ کاؤنٹی، ورجینیا"
     ],
-    "name:urd_x_variant":[
-        "\u0628\u06cc\u0688\u0641\u0648\u0631\u0688 \u06a9\u0627\u0624\u0646\u0679\u06cc"
+    "name:urd_x_variant": [
+      "بیڈفورڈ کاؤنٹی"
     ],
-    "name:vie_x_preferred":[
-        "Bedford"
+    "name:vie_x_preferred": [
+      "Bedford"
     ],
-    "name:vie_x_variant":[
-        "Qu\u1eadn Bedford"
+    "name:vie_x_variant": [
+      "Quận Bedford"
     ],
-    "name:war_x_preferred":[
-        "Condado han Bedford"
+    "name:war_x_preferred": [
+      "Condado han Bedford"
     ],
-    "name:zho_x_preferred":[
-        "\u8c9d\u5fb7\u798f\u7e23"
+    "name:zho_x_preferred": [
+      "貝德福縣"
     ],
-    "qs:a0":"United States",
-    "qs:a1":"Virginia",
-    "qs:a1_lc":"51",
-    "qs:a2":"Bedford",
-    "qs:a2_alt":"Bedford County",
-    "qs:a2_lc":"51019",
-    "qs:adm0":"United States",
-    "qs:adm0_a3":"USA",
-    "qs:level":"adm2",
-    "qs:source":"US Census",
-    "src:geom":"quattroshapes",
-    "src:geom_alt":[],
-    "src:lbl_centroid":"mapshaper",
-    "src:population":"uscensus",
-    "src:population_date":"2023",
-    "statoids:area_km":"1968",
-    "statoids:as_of_date":"2010-04-01",
-    "statoids:capital":"Bedford",
-    "statoids:country":"US",
-    "statoids:fips":"51019",
-    "statoids:hasc":"US.VA",
-    "statoids:name":"Bedford",
-    "statoids:population":"74898",
-    "statoids:statoid":"US.VA.BF",
-    "statoids:timezone":"-5~",
-    "statoids:type":"county",
-    "uscensus:population":81525,
-    "uscensus:population_date":"2023",
-    "wof:belongsto":[
-        102191575,
-        85633793,
-        85688747
-    ],
-    "wof:breaches":[],
-    "wof:concordances":{
-        "fips:code":"51019",
-        "gp:id":12590314,
-        "hasc:id":"US.VA.BF",
-        "uscensus:geoid":"51019",
-        "wd:id":"Q494216"
+    "qs:a0": "United States",
+    "qs:a1": "Virginia",
+    "qs:a1_lc": "51",
+    "qs:a2": "Bedford",
+    "qs:a2_alt": "Bedford County",
+    "qs:a2_lc": "51019",
+    "qs:adm0": "United States",
+    "qs:adm0_a3": "USA",
+    "qs:level": "adm2",
+    "qs:source": "US Census",
+    "src:geom": "uscensus",
+    "src:geom_alt": [
```

Comment:
For records with new alt geometries, we should also add a `wof:geom_alt` property. https://github.com/whosonfirst/whosonfirst-properties/tree/main/properties/wof#geom_alt

It doesn't look like any of these new alt files have {extras} in the file name, so I think that would just mean:

```
"wof:geom_alt":[
  "quattroshapes"
],
```

Reply:
Fixed in [cfcfe4e](https://github.com/whosonfirst-data/whosonfirst-data-admin-us/pull/207/commits/cfcfe4edcec60eda0e042663784a751b5d537574)